workshop/content/60-go/20-create-project/300-structure.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "Project structure"
+weight = 300
++++
+
+## Open your IDE
+
+Now's a good time to open the project in your favorite IDE and explore.
+
+> If you use VSCode, you can just type `code .` within the project directory.
+
+## Explore your project directory
+
+You'll see something like this:
+
+![](./structure.png)
```

Comment:
Add alt-text, even if it is "Image of the list of files described after this image".

Reply:
I created an [FR for alt text](https://github.com/aws-samples/aws-cdk-intro-workshop/issues/648), we should handle this in a separate PR once Go is up